README.md

```diff
@@ -167,9 +167,9 @@ load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
 
 container_deps()
 
-load("@io_bazel_rules_docker//repositories:pip_repositories.bzl", "pip_deps")
+load("@io_bazel_rules_docker//repositories:pip_repositories.bzl", "io_bazel_rules_docker_pip_deps")
```

Comment:
This snippet is using `rules_docker-0.14.4` which doesn't have your rename so copy and pasting now fails.

```
ERROR: Traceback (most recent call last):
        File "/usr/local/google/home/mikedanese/code/cloud-provider-gcp/WORKSPACE", line 70, column 89, in <toplevel>
                load("@io_bazel_rules_docker//repositories:pip_repositories.bzl", container_pip_deps = "io_bazel_rules_docker_pip_deps")
Error: file '@io_bazel_rules_docker//repositories:pip_repositories.bzl' does not contain symbol 'io_bazel_rules_docker_pip_deps'
ERROR: error loading package 'external': Package 'external' contains errors
```

Reply:
@smukherj1 and @aaliddell Tagging another release seems like a good idea, and the README.md could be updated at the same time.

Reply:
Ack. Made a note of this and will get around to this in the next week or so hopefully.

Reply:
friendly ping :)